ws/src/speech/scripts/Whisper.py

```diff
@@ -6,75 +6,74 @@
 
 '''
 
-# TODO: 
-# Reduce audio messages in pipeline, while maintaining decoupling 
-# Check which whisper model performs better
 
 import rospy
 import whisper
 import torch
-
+import os
 from WavUtils import WavUtils
 
 from audio_common_msgs.msg import AudioData
-from speech.msg import RawInput
 from std_msgs.msg import String
 from frida_hri_interfaces.srv import AudioText, AudioTextResponse
 
-#SPEECH_COMMAND_TOPIC = "RawInput"
 SPEECH_COMMAND_TOPIC = "/speech/raw_command"
 SPEECH_SERVICE_TOPIC = "/speech/service/raw_command"
 DEBUG = True
 
+CURRENT_FILE_PATH = os.path.abspath(__file__)
+CURRENT_DIRECTORY = os.path.dirname(CURRENT_FILE_PATH)
+MODEL_DIRECTORY = os.path.join(CURRENT_DIRECTORY, 'models')
```

Comment:
exits check?

Reply:
?

Reply:
Maybe there is an edge case but __file__ should (almost always) exist. Model directory is created by `whisper._download` if it doesn't exist.